Guardfile

```diff
@@ -1,13 +1,13 @@
 rspec_options = {
   version:        1,
   all_after_pass: false,
-  all_on_start:   false,
```

Comment:
I actually prefer the trailing comma. Having it means never needing to remember it when you add a new entry and it also keeps it from showing up in the git history.